### PR TITLE
improvement to x content headers passed to oproxy

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description 
x-content headers to use utility function from common server python

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests - unit tests
- [ ] Documentation 

## Additional
* Also added the following utility function: `get_demisto_version_as_str` to CommonServerPython
* As a result of running demisto-sdk format description fields of yml files have line spacing modifications.

### CODE REVIEW

File: Packs/MicrosoftGraphFiles/pack_metadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    "name": "Microsoft Graph Files",
+    "description": "Use the Microsoft Graph Files integration to enable your app get authorized access to files in OneDrive, SharePoint, and MS Teams across your entire organization. This integration requires admin consent.",
+    "support": "xsoar",
+    "currentVersion": "1.0.1",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "created": "2020-04-14T00:00:00Z",
+    "categories": [
+        "Authentication"
```
Comment (Itay4) [inline Packs/MicrosoftGraphFiles/pack_metadata.json:11]: please update here and in the integration yml the category

**Comment by Itay4** [comment]
what about azure sentinel ?

**Comment by glicht** [comment]
> what about azure sentinel ?

Seems there is some bug in demisto-sdk format that ignores this file. @roysagi is checking

**Comment by roysagi** [comment]
@glicht @Itay4 - Format command currently does not support beta integrations - hence the observed behavior.
This is not a bug, but a future enhancement we should make.

File: Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml [hunk 2]
```diff
@@ -178,8 +178,8 @@ script:
       required: true
       secret: false
     - default: false
-      description: 'The unique ID of the mail. You cannot use the the ''MessageID'' key in the
-        form ''<message-id>''.'
+      description: 'The unique ID of the mail. You cannot use the the ''MessageID''
```
[the remark this answers is not in the document]
Comment (glicht) [reply Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml:181]: I ran format to update the docker image and it updates also long strings.

File: Packs/MicrosoftGraphFiles/pack_metadata.json (hunk 1 above)
Comment (glicht) [reply Packs/MicrosoftGraphFiles/pack_metadata.json:11]: will update to "IT Services"

**Review by Itay4 — APPROVED** [review]

**Review by anara123 — APPROVED** [review]

**Comment by glicht** [comment]
force merge as unrelated failures.

**Pull request merged**